packages/ses/test/tame-legacy-regenerator-helper.test.js

```diff
@@ -0,0 +1,53 @@
+import test from 'ava';
+import '../index.js';
+
+lockdown({ legacyRegeneratorRuntimeTaming: 'unsafe-ignore' });
+
+test('lockdown Iterator.prototype[@@iterator] is tamed', t => {
+  const IteratorProto = Object.getPrototypeOf(
+    Object.getPrototypeOf([].values()),
+  );
+  const desc = Object.getOwnPropertyDescriptor(IteratorProto, Symbol.iterator);
+  if (!desc || !desc.get || !desc.set) throw new Error('unreachable');
+  t.is(desc.configurable || desc.enumerable, false);
+  t.is(desc.value, undefined);
+
+  const { get, set } = desc;
+  t.is(
+    Function.prototype.toString.call(get),
+    'function get() { [native code] }',
+  );
+  t.is(
+    Function.prototype.toString.call(set),
+    'function set() { [native code] }',
+  );
+
+  const child = Object.create(IteratorProto);
+  child[Symbol.iterator] = 'foo'; // override test
+  t.is(child[Symbol.iterator], 'foo');
+
+  const native = get();
```

Comment:
Thanks for testing this. It surprised me that this worked. But then I looked at the getter installed by `enable-property-overrides.js` and realized that it would work for those too. So no change suggested.